### PR TITLE
A couple typo fixes in the help files

### PR DESCRIPTION
**Pull request by erinann**
- Fixed the intro sentence in the Brackets file
- Fixed an image link in the Brackets file

### CODE REVIEW

**Comment by gavinmacaulay** [comment]
The PR has changes in files that don't relate to your intended changes (e.g., the automatically generated .html files).  Does anyone know of a way to ignore changes in these auto-generated files, but still have them in the repo? (or some other solution that achieves the same).

**Review by gavinmacaulay — COMMENTED** [review]
Was the Testing heading in acmeta.adoc intended?
We also need to come up with a plan to avoid the autogenerated and committed .html files from appearing commits...

**Comment by erinann** [comment]
@gavinmacaulay ,

Good question. I've been wondering about any issues with the autogenerated files. PRs merge all commits (that I can tell). I'll look into this issue. 

No, the Testing header was not intended. I had a merge conflict that I might not have addressed correctly and did not realized that was added back in.

File: AcMeta/acmeta.adoc [hunk 1]
```diff
@@ -31,6 +31,8 @@ ICES. 2020. A metadata convention for processed acoustic data from active acoust
 systems. Series of ICES Survey Protocols SISP 4-TG-AcMeta. 48 pp.
 // [Tim R: May need to update this citation when this git document gets published by ICES as part of the SISP series]
 
+*Testing*
+
```
Comment (TimRyan99) [inline AcMeta/acmeta.adoc:35]: not sure what this Testing line is about. can you clarify

Comment (ghost) [reply AcMeta/acmeta.adoc:35]: Hi @TimRyan99 , please ignore this pull request. I think Gavin should close it without merging. I was updating the help files and inadvertently changed the auto-generated AcMeta doc when I was testing some changes. It is not a good pull request. 

Let me know if you have any questions. 

... and I just realized that I'm logged in with my testing account. It's Erin (@erinann). Please ignore this pull request or close it. Thanks.

Comment (TimRyan99) [reply AcMeta/acmeta.adoc:35]: ok, finding my way around the approval/merge process. I can close this pull request so will do so. cheers

**Comment by TimRyan99** [comment]
Following advice from @erinann, closing pull request.
Hi @TimRyan99 , please ignore this pull request. I think Gavin should close it without merging. I was updating the help files and inadvertently changed the auto-generated AcMeta doc when I was testing some changes. It is not a good pull request

**Pull request closed without merging**